src/lamp_py/bus_performance_manager/events_gtfs_schedule.py

```diff
@@ -339,7 +339,17 @@ def bus_gtfs_schedule_events_for_date(service_date: date) -> pl.DataFrame:
         "departure_seconds",
         "parent_station",
     ]
-    stop_events = stop_events.drop(drop_columns).rename({"trip_id": "plan_trip_id"})
+    stop_events = (
+        stop_events.drop(drop_columns)
+        .rename({"trip_id": "plan_trip_id"})
+        .with_columns(
+            pl.col("plan_trip_id")
+            .str.replace(r"_\d", "")
```

Comment:
why not call `remove_overload_and_special_route_suffix` so that logic is centrally editable?